config/foreman.hiera/common.yaml

```
@@ -9,3 +9,5 @@ dhcp::config_comment: |
   foreman-installer. Use custom-hiera.yaml for multiple subnets.
 
 mongodb::server::set_parameter: "{diagnosticDataCollectionEnabled: false}"
+
+foreman::config::apache::proxy_params: {'retry': '0', 'timeout': '900'}
```

Why do you use this syntax? Doesn't this work as well?
```suggestion
foreman::config::apache::proxy_params:
  retry: '0'
  'timeout': '900'
```

Btw, not sure why we quoted the value in the Puppet manifest. Not sure that's really needed.

---

YAMl vs Ruby brain. I updated to this cleaner style.